src/metaform-react/MetaformEditableHtmlComponent.tsx

```diff
@@ -0,0 +1,88 @@
+import { Box } from "@mui/material";
+import React from "react";
+import ReactQuill from "react-quill";
+import { MetaformField } from "../generated/client/models";
+import "react-quill/dist/quill.snow.css";
+
+/**
+ * Component props
+ */
+interface Props {
+  field: MetaformField;
+  onFieldUpdate: (updatedField: MetaformField) => void;
+}
+
+/**
+ * Component for Metaform editable html field
+ *
+ * @param props component properties
+ */
+export const MetaformEditableHtmlComponent: React.FC<Props> = ({
+  field,
+  onFieldUpdate
+}) => {
+  if (!field.name) {
+    return null;
+  }
+
+  /**
+   * On field html update
+   *
+   * @param updatedHtml updated html
+   */
+  const onFieldHtmlUpdate = (updatedHtml: string) => {
+    console.log(updatedHtml);
+    onFieldUpdate({
+      ...field,
+      html: updatedHtml
+    });
+  };
+
+  const modules = {
+    toolbar: {
+      container: [
+        ["bold", "italic", "underline", "strike", "blockquote"],
+        ["headingOne", "headingTwo", "headingThree"],
```

Comment:
Should this be heading choices?  Seems that it's not working properly.
Try this [{ 'header': [1, 2, 3, 4, 5, 6, false] }],

Also we could add text color property here also.

Reply:
Was just testing it and I can confirm that works and correctly applies a dropdown menu for different header options.
```suggestion
        [{ "header": [ 1, 2, 3, 4, 5, 6, false ] }],
```